services/resource-usage-tracker/src/simcore_service_resource_usage_tracker/models/resource_tracker_container.py

```diff
@@ -26,17 +27,18 @@ class ContainerScrapedResourceUsageMetric(BaseModel):
         None,
         description="Instance label scraped via Prometheus (taken from container labels, ex.: gpu1)",
     )
-    service_settings_reservation_nano_cpus: int | None = Field(
-        None,
-        description="CPU resource limit allocated to a container, ex.500000000 means that the container has limit for 0.5 CPU shares",
-    )
-    service_settings_reservation_memory_bytes: int | None
     service_settings_reservation_additional_info: dict[str, Any] = Field(
         {},
         description="Storing additional information about the reservation settings, such as what type of graphic card is used.",
     )
-    service_settings_limit_nano_cpus: int | None
-    service_settings_limit_memory_bytes: int | None
+    memory_limit: int = Field(
```

Comment:
have you considered this constraint type https://docs.pydantic.dev/1.10/usage/types/#bytesize for this?

Reply:
no I have not, thank you I will use it :+1: